scripts/server

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash 
+
+script_dir=$(dirname "$0") && cd $script_dir && script_dir=$(pwd) && source ./_helpers
+# requireDependency 'dotnet' '1.0.4'
+requireDependency 'npm'
+
+# TODO: clarify intent for serving/hosting app - locally it appears to work fine from frotnend only
```

Comment:
It works fine locally for now because the ip for the api is hardcoded to that azure hosted instance.   I need to get around to having it hit the local api build as it seems you were trying to do here.

Reply:
@jopache I've since addressed some of this for ya to help out. 

- added multiple environments (https://github.com/jopache/Settings/pull/18/commits/e120c034467b3922be16ee6234c7f9e40ce577cc, https://github.com/jopache/Settings/pull/18/commits/08072943089445c6c58f865f00ae4cd983f1ef1c)
- updated server script to spawn frontend/backend (https://github.com/jopache/Settings/pull/18/commits/4abe489878185e19c1f4c7543e26fba454b6d5a9)

Reply:
<3